fe/lib/components/LocationSuggest/LocationSuggestInput.tsx

```diff
@@ -153,6 +174,41 @@ const LocationSuggestInput = ({
           value={locationSuggest}
         />
       </Column>
+      <Column width="content">
+        <Button
+          icon={<IconLanguage />}
+          onClick={() => setShowMapDialog(true)}
+          tone={tone === 'critical' ? tone : undefined}
+          variant="soft"
+        >
+          Select on map
+        </Button>
+        {showMapDialog ? (
+          <Dialog
+            id="map-dialog"
+            title="Select a location"
+            width="medium"
+            description={
+              <Text tone="secondary">
+                Select a point on the map to choose from a list of{' '}
+                <SmartTextLink href="https://developer.seek.com/use-cases/job-posting/locations#nearestlocations">
+                  nearest locations
+                </SmartTextLink>
+              </Text>
+            }
+            open={showMapDialog}
+            onClose={() => {
+              setShowMapDialog(false);
+            }}
+          >
+            <MapSelect
+              schemeId={schemeId}
+              onLocationSelected={handleMapLocationSelected}
+              initialLocation={SEEK_MEL_CENTER}
```

Comment:
I think it may make more sense to embed this default in the `MapSelect`, unless you were thinking of allowing consumers to override the initial location

Reply:
Yeah I did this with that idea in mind, e.g. might want to dynamically set the location depending on what region the user is viewing from. Probably ties into [this](https://github.com/seek-oss/wingman/pull/834#discussion_r996506967) comment below from @72636c

Reply:
Scope creep, but this now makes me wonder whether there should be some interplay with the "detect location" feature. I imagine it may make sense for `initialLocation` to start at the current location of the user.

Reply:
@72636c Agreed, I have made it so that if a detected location exists it will centre on that instead of SEEK HQ 😄